cmd/bbolt/main.go

```diff
@@ -1095,6 +1098,8 @@ func (cmd *benchCommand) ParseFlags(args []string) (*BenchOptions, error) {
 	fs.BoolVar(&options.Work, "work", false, "")
 	fs.StringVar(&options.Path, "path", "", "")
 	fs.BoolVar(&options.GoBenchOutput, "gobench-output", false, "")
+	fs.IntVar(&options.PageSize, "page-size", common.DefaultPageSize, "Set page size in bytes.")
```

Comment:
setting default page to 0 should be good enough, as bbolt will automatically select `common.DefaultPageSize` internally. But not a big problem